wellknownimports/wellknownimports.go

```diff
@@ -18,14 +18,57 @@ package wellknownimports
 
 import (
 	"embed"
+	"fmt"
 	"io"
+	"io/fs"
+	"path/filepath"
 
 	"github.com/bufbuild/protocompile"
 )
 
 //go:embed google/protobuf/*.proto google/protobuf/*/*.proto
 var files embed.FS
 
+var filesByName = make(map[string]string)
+
+func init() {
+	// We initialize a map instead of going through the embed.FS for two reasons:
+	//
+	// 1. ReadFile performs an unnecessary copy of the file's text.
+	// 2. It requires some non-trivial work to identify when the user passes a
+	//    directory such as "google/" or "google/protobuf/".
+	err := fs.WalkDir(files, ".", func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+
+		if !d.IsDir() {
+			bytes, err := files.ReadFile(path)
+			if err != nil {
+				return err
+			}
+
+			// Technically ToSlash is redundant here, since embed.FS always uses
+			// / as a separator. But it's a good idea to avoid surprises.
```

Comment:
I don't understand this comment. `ToSlash` is **not** redundant here because you use `filepath.Clean` not `path.Clean`. But why do you need to clean the path?

Reply:
This function is no longer used so I dropped it.